### PR TITLE
bzflag: update to version 2.4.14

### PR DESCRIPTION
**Pull request by allejo**
#### Description

Hi, macports team! I'm gonna need your help on this one since I've never contributed on here. I'm one of the developers for BZFlag and saw that there wasn't a maintainer for the macports version of BZFlag so I figured I'd take a stab at it.

Let me know what I did wrong and/or need to do correctly.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.10.4 14E46
Xcode 6.4 6E35b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: games/bzflag/Portfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ PortSystem          1.0
 PortGroup           app 1.0
 
 name                bzflag
-version             2.4.12
+version             2.4.14
 categories          games
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @allejo
```
Comment (pmetzger) [inline games/bzflag/Portfile]: You need to list both your email address and your github handle here, in braces. See other ports for examples, or the guide. Also, I would recommend that, unless there's a good reason not to, you also set `openmaintainer` to indicate you don't mind minor changes being made by MacPorts team members.

File: games/bzflag/Portfile [hunk 2]
```diff
@@ -18,8 +18,8 @@ homepage            https://www.bzflag.org/
 master_sites        https://download.bzflag.org/bzflag/source/${version}/
 use_bzip2           yes
 
-checksums           rmd160 6ea084cdcded82cc63dc9a0455bb631ad6454c56 \
-                    sha256 a8a13a58648798a6f6f969a2911cf21377fe45964f2edf46a327d1370ff1000d
+checksums           rmd160 4831f3915256fcb8c762acbd47bcc19afdaeb02c \
+                    sha256 09a46ec14cf956cb69fd069cee978de4dec321cb563036b16367ea8aae529bdc
```
Comment (pmetzger) [inline games/bzflag/Portfile]: These days, we also have a `size` field that typically gets added after the `sha256` field. You can generate all these automatically with `port -v checksum`, fyi.

Comment (allejo) [reply games/bzflag/Portfile]: Thank you! That's a convenient command to know. I was just using `openssl` 😅

File: games/bzflag/Portfile (hunk 1 above)
Comment (allejo) [reply games/bzflag/Portfile]: Added my email and `openmaintainer` as well

**Comment by pmetzger** [comment]
Merged. Thank you, @allejo!

**Pull request merged**